### PR TITLE
Bring formatting up to scratch.

### PR DESCRIPTION
**Pull request by pthariensflame**

### CODE REVIEW

File: SwiftxTests/EitherSpec.swift [hunk 1]
```diff
@@ -15,9 +15,9 @@ extension Either where L : Arbitrary, R : Arbitrary {
 		return Gen.oneOf([
 			Either.Left <^> L.arbitrary,
 			Either.Right <^> R.arbitrary,
-		])
+			])
```
Comment (CodaFi) [inline SwiftxTests/EitherSpec.swift]: [This is Xcode's fault] but put this back.

File: SwiftxTests/SectionSpec.swift [hunk 2]
```diff
@@ -17,117 +17,117 @@ class SectionTests: XCTestCase {
...
 		}
-		
+
 		property("") <- forAll { (x : ArrayOf<Int>, i : Int) in
 			let xs = x.getArray
 			return
 				xs.map(|i) == xs.map { $0 | i }
-				^&&^
-				xs.map(i|) == xs.map { i | $0 }
+					^&&^
```
Comment (CodaFi) [inline SwiftxTests/SectionSpec.swift]: Why are all of these unaligned?

**Comment by CodaFi** [comment]
Can you remove the version variables from our public header?

**Comment by pthariensflame** [comment]
Ok!

**Comment by CodaFi** [comment]
All clear

**Comment by CodaFi** [comment]
:boat:

**Pull request merged**